Cineaste/Extension/Date+Formatter.swift

```diff
@@ -9,15 +9,24 @@
 import UIKit
 
 extension Date {
-    var formatted: String {
+    private static let dateFormatter = { () -> DateFormatter in
         let formatter = DateFormatter()
-        formatter.dateFormat = "dd.MM.YYYY"
-        return formatter.string(from: self)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        return formatter
+    }()
+
+    var formatted: String {
+        Date.dateFormatter.dateFormat = "dd.MM.YYYY"
+        return Date.dateFormatter.string(from: self)
     }
 
     var formattedForRequest: String {
-        let formatter = DateFormatter()
-        formatter.dateFormat = "yyyy-MM-dd"
-        return formatter.string(from: self)
+        Date.dateFormatter.dateFormat = "yyyy-MM-dd"
```

Comment:
Same here as above

Reply:
As we have to format the date for the request independently from the locale, I don't think it is a good idea to use some pre-defined date styles as the style may change in the future (who knows). I would use the specific date format in this case.